sdk/core/test/integration/otp/core.test.ts

```diff
@@ -733,5 +733,20 @@ parallel('CommonNetworkMember [OTP]', () => {
 //       const { wallet } = await AffinidiWallet.signInWithProfile(options, trueCallerTestProfile)
 //       checkIsWallet(wallet)
 //     })
+//
```

Comment:
why it is commented?

Reply:
it is temporarily blocked by infrastructure issues. will be uncommented after it will be resolved

Reply:
if it is temp, i think we should use it.skip()